.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.10"]
```

Comment:
Let's make sure this does not affect the PR across forks! We want to keep more testing on the main fork, but want to avoid exceeding the github basic quota on @ndickson-nvidia 's github